vendor/assets/stylesheets/dvl/components/sidebar_sub_actions.scss

```diff
@@ -5,5 +5,8 @@
   font-size: $fontSmall;
   .sidebar_box + & {
```

Comment:
We always align `.sidebar_box` to the right and `.sidebar_nav` to the left, so the styles here match the intent of the design. Not a hack.